internal/cluster/cluster_operations.go

```diff
@@ -402,6 +401,9 @@ func ImportCluster(ctx context.Context, data *schema.ResourceData, apiClient *cl
 	_ = data.Set("is_default", clusterObj.IsDefault)
 	_ = data.Set("is_stretched", clusterObj.IsStretched)
 	flattenedVdsSpecs := getFlattenedVdsSpecsForRefs(clusterObj.VdsSpecs)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Why is this error check added here? err variable is last set at line 393.

Best practice is to return the error and exit the function immediately, not 10 instructions later.